### PR TITLE
Add Kochi to Json File

### PR DESCRIPTION
**Pull request by picklehari**

### CODE REVIEW

File: data/chapters.json [hunk 1]
```diff
@@ -27,6 +27,13 @@
     },
     "title": "St. Louis"
   },
+,
```
Comment (DarrenN) [inline data/chapters.json]: Looks like you have a stray comma here.

File: data/chapters.json [hunk 2]
```diff
@@ -27,6 +27,13 @@
...
+  "kochi": {
+    "dataService": {
+      "id": "papers-we-love-kochi",
+    },
```
Comment (DarrenN) [inline data/chapters.json]: Also, are you using Meetup.com?

Comment (picklehari) [reply data/chapters.json]: Presently, no. Given the circle within Kochi. Most event publicity happens via telegram. However, we just have had three meetups so far. We'll set it up soon if we are to.

File: data/chapters.json (hunk 1 above)
Comment (DarrenN) [reply data/chapters.json]: You still have a stray comma, I can't merge this until its corrected.

**Pull request closed without merging**